### ISSUE TITLE
Bug Report: When minimizing the window (and therefore pausing), Discord rich presence timer desyncs

### ISSUE THREAD

**Issue by actuallyasmartname** [KadeArchive/Kade-Engine#281]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x64`)

## What version of the game are you using? Look in the bottom left corner of the main menu.
Funkin 0.2.7.1, KE 1.4.2
## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.
1. Open Kade Engine and Discord
2. Play a song (anything)
3. Minimize the game and look at Discord
4. Discord's  "Time Left" still counts down, only to return to it's previous state when the window is not minimized
## Please describe your issue. Provide extensive detail and images if possible.
So basically if you minimize the window, the game "pauses".
If you reopen it goes back to where you were.
Now what I expected was Discord to say at least "Game paused" or some shit like that however it doesn't.
It simply counts down to 0:00 and just stops there unless you reopen the window and the "timer" is back to where it was.

**Comment by puyoxyz** [comment]
Renamed: `Bug Report: When minimizing the window (and therefore pausing), Discord rich presence timer desyncs`

**Comment by actuallyasmartname** [comment]
Thank you.
